webpack.config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
   },
   module: {
     loaders: [{
+      test: /\.js$/,
+      exclude: /(node_modules|bower_components)/,
```

Comment:
IMO, ES6 is not used in all projects, so most of packages should compatible with ES5.
That said, in most cases, there is no need to include third party packages into babel process.